x-pack/plugins/security_solution/common/api/timeline/model/components.gen.ts

```diff
@@ -306,6 +357,11 @@ export const SortFieldTimeline = z.enum(['title', 'description', 'updated', 'cre
 export type SortFieldTimelineEnum = typeof SortFieldTimeline.enum;
 export const SortFieldTimelineEnum = SortFieldTimeline.enum;
 
+export type SortDirection = z.infer<typeof SortDirection>;
```

Comment:
I see your point but it would mean rename a bunch more types in this file and then all over security solution. We could think of, in a future PR, to streamline usage of these types. I see occurrences of `asc` and `desc` and enums for these, all over security solution, not just in timeline.